strapi/strapi
# ML: Unable to select all media library assets, when page is greater than 1

### What does it do?

Fixes the issue with the inability to select all `Media Library `assets, when the page is greater than 1.

### Why is it needed?
When we attempt to select all in a media library with no folders, the `folder` variable being `undefined`, throws an error when we try to access `folders.map` 

### How to test it?

1. Go to Media Library
2. Upload a couple of assets such that you have multiple pages
3. Try to use the select all checkbox on the second or any page number which isn't 1
4. Verify that it selects all the items?

 
### Demos

https://user-images.githubusercontent.com/7220167/206154858-894ca9bb-525a-411e-87a3-d26acd41f14d.mov


### Related issue(s)/PR(s)

- Fixes https://github.com/strapi/strapi/issues/15062
